nixos/modules/services/web-servers/keter/bundle.nix

```diff
@@ -0,0 +1,35 @@
+/* This makes a keter bundle as described on the github page:
+   https://github.com/snoyberg/keter#bundling-your-app-for-keter
+
+   Note that we're not actually putting in the app into the bundle,
+   we already can use the nix store for copying, so we just
+   symlink to the app.
+*/
+{ domain,
+  exePkg,
+  executable, gnutar, writeTextFile, lib, stdenv, ... }:
+
+let
+  str.stanzas = [{
+    # we just use nix as an absolute path so we're not bundling any binaries
+    type = "webapp";
+    exec = "${exePkg}/bin/${executable}";
```

Comment:
Please just take in one `types.path` option instead of two options for the same thing.

Reply:
I did this and removed the default, took me a while to figure out how to get the tests to work because the new keter version has better error handling.